packages/std/src/ibc.rs

```diff
@@ -215,6 +213,17 @@ impl IbcAcknowledgement {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct IbcChannelOpenMsg {
     pub channel: IbcChannel,
+    pub variant: IbcChannelOpenVariant,
+}
+
+/// This differentiates between IbcChannelOpen and IbcChannelTry
+#[serde(rename_all = "snake_case")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub enum IbcChannelOpenVariant {
+    Open {},
+    Try {
+        counterparty_version: Option<String>,
```

Comment:
In this case, it is always set, right?

```suggestion
        counterparty_version: String,
```

Reply:
Yup just fixed that with a new commit